packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1522,6 +1522,14 @@ class _RendersOnPhysicalModel extends _MatchRenderObject<RenderPhysicalShape, Re
       return false;
     }
 
+    if (
+      borderRadius == null &&
+      (shape == BoxShape.oval || shape == BoxShape.oval) &&
+      !assertOval(shapeClipper, matchState)
+    ) {
```

Comment:
Formatting here is a little different from what we normally do:

```suggestion
    if (borderRadius == null &&
        (shape == BoxShape.oval || shape == BoxShape.oval) &&
        !assertOval(shapeClipper, matchState)) {
```

Reply:
<img width="710" alt="image" src="https://user-images.githubusercontent.com/351125/178827823-be18c27c-75a9-4692-9251-2f38e86d044a.png">

yes, but that was the reason lol

Reply:
Ahh, OK.  Well, we don't usually format it like that.

Reply:
Fixed the others in https://github.com/flutter/flutter/pull/103833/commits/6cfd0938e0e5434bd052f784ee9953f09a0881db